.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+max-line-length = 100
+exclude=
+    tokenizer/python/input.py
+
+per-file-ignores =
+    tokenizer/mips/mips_tokenizer.py:W605
```

Comment:
What lines are failing this check? Wondering how hard to fix, as looking it up, looks like this warning will become a `SyntaxError` soon?

Reply:
The issue is the `\d`\ on line 17 I believe.  I'm not the best with regex so I left it in the code but I agree that it should ideally be fixed.